src/lib/Libutil/misc_utils.c

```diff
@@ -1782,3 +1782,59 @@ perf_stat_stop(char *instance)
 
 	return (stat_summary);
 }
+
+/**
+ * @brief
+ *	creates an empty file in /tmp/ and saves timestamp of that file
+ *
+ * @param[in] - void
+ *
+ * @return - void
+ */
+void
+create_query_file(void)
+{
+	char filename[30];
+	char command[30];
+	snprintf(filename, 30, "/tmp/last_query_%d", getuid());
+#ifdef WIN32
+	snprintf(command, 30, "type nul > %d", filename);
+#else
+	snprintf(command, 30, "touch %s", filename);
+#endif
+	system(command);
```

Comment:
It's cheaper to call open(2)/fopen(3) than it is to call system(3). If you want to guarantee you have fine granularity in the value you should write the timestamp into the file as a floating point value rather than relying on the underlying filesystem.